### PR TITLE
[TEVA-1089] Add job listing banner

### PR DESCRIPTION
**Pull request by cpjmcquillan**
This PR adds a grey full width banner to the job listings page.

## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1089

## Changes in this PR:
- Add full width banner
- Remove padding on vacancies_show and vacancies_preview pages

## Screenshots of UI changes:
image

### CODE REVIEW

File: app/frontend/styles/components/hiring-staff.scss [hunk 1]
```diff
@@ -22,7 +22,6 @@ body.hiring-staff {
     .govuk-main-wrapper {
       margin-left: auto;
       margin-right: auto;
-      max-width: 960px;
```
Comment (david-mears-dfe) [inline app/frontend/styles/components/hiring-staff.scss:25]: @josephhull676 NB. We should rebase on master after this is merged.

File: app/frontend/styles/components/jobs-banner.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.jobs-banner {
+  background-color: govuk-colour('light-grey');
+  left: 50%;
```
Comment (david-mears-dfe) [inline app/frontend/styles/components/jobs-banner.scss:3]: > If position: relative, the left property sets the left edge of an element to a unit to the left/right of its normal position.

Don't the left and right properties cancel out?

Comment (cpjmcquillan) [reply app/frontend/styles/components/jobs-banner.scss:3]: I think we only need left/margin-left properties and can forgo the right ones

**Review by david-mears-dfe — APPROVED** [review]

File: app/frontend/styles/components/jobs-banner.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  margin-bottom: govuk-spacing(3);
+  margin-left: -50vw;
+  margin-right: -50vw;
```
Comment (alexbowen) [inline app/frontend/styles/components/jobs-banner.scss:6]: usually i like to put it all in one line if there are more than 2 properties `margin: 0 -50vw govuk-spacing(3) -50vw` but no biggie.

Comment (cpjmcquillan) [reply app/frontend/styles/components/jobs-banner.scss:6]: I'll try it out on my other branch (I just forget every time where it starts and which direction it goes 😂 )

**Review by alexbowen — APPROVED** [review]

**Pull request merged**